### PR TITLE
Add new airflow logo to orbit to replace airflow with/without stars logo throughout app

### PR DESCRIPTION
**Pull request by petedejoy**
This PR replaces all of our "yellow-finned" Airflow logos (with and without stars) with the latest Airflow logo. I consulted with @Inkdryer and we decided to move away from the "with stars" logos throughout Orbit.

### CODE REVIEW

**Comment by schnie** [comment]
@petedejoy you figure out how to shrink the logo at all?

**Comment by petedejoy** [comment]
I struggled a bit. @samblackk do you have tips on how to shrink SVGs? It looks like they're all styled to the same height/width in orbit, but most of the others have some padding so the new Airflow logo looks a bit too big. Seems like it'd be easier to do in the SVG file itself rather than apply custom styles from a higher level to this single icon, but not sure what the best way to do that is.

File: src/instruments/icons/Icon/img/airflow-new.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="150pt" height="150pt" viewBox="0 0 150 150" version="1.1">
```
Comment (mik-laj) [inline src/instruments/icons/Icon/img/airflow-new.svg]: I think that this graphic should be saved in px resolution

**Comment by mik-laj** [comment]
If you want to change add padding then you can easily do it by editing the viewBox in the file.

**Comment by samblackk** [comment]
@petedejoy I adjusted the logo's whitespace & their sizing throughout. Verify it's what you're going for.

**Comment by samblackk** [comment]
Screen Shot 2019-10-02 at 11 05 39 AM

**Comment by petedejoy** [comment]
Ok- looking better now across the board: 

image

Thanks for the help here, @samblackk!

**Comment by mik-laj** [comment]
Do you have a gray variant of the Airflow logo? Do you need it? I am working on a brandbook and I have this version, but it has not been approved by the community.

**Comment by schnie** [comment]
cc: @Inkdryer for gray version :)

**Comment by schnie** [comment]
This is merged. @Inkdryer if you want to make any changes let us know.

**Pull request merged**